.gitignore

```diff
@@ -0,0 +1,10 @@
+.idea
```

Comment:
Follow these steps:

- git checkout main
- git pull
- git checkout t2
- git merge main
- git push


This will put the latest code from main branch in the t2 branch


when you do these steps, look at this PR. you will notice lots of these files would have gone missing

Reply:
nothing to see missing .  i get this message (everything is up to date)